ppr-ui/tests/unit/TableRow.spec.ts

```diff
@@ -208,7 +208,7 @@ describe('TableRow tests', () => {
         expect(rowData.exists()).toBe(true)
         expect(rowData.length).toBe(11)
         // reg num
-        if (isChild) expect(rowData.at(0).text()).toBe(`Pending  Base Registration: ${baseReg.baseRegistrationNumber}`)
+        if (isChild) expect(rowData.at(0).text()).toBe(`Pending   Base Registration: ${baseReg.baseRegistrationNumber}`)
```

Comment:
Additional spacing intentional?

Reply:
Yes... It takes care of the line break I added. 
The line break <br>was added to ensure there is a line break for both Ppr and registration number wraps.

Reply:
Sounds good, thanks!